src/TokenRepository.php

```diff
@@ -54,6 +54,8 @@ public function forUser($userId)
     /**
      * Get a valid token instance for the given user and client.
      *
+     * @deprecated use findValidToken
```

Comment:
There's no `findValidToken` on this class? Did you want to reference a different class?

Reply:
https://github.com/laravel/passport/blob/22b3861eafce8c2cde78d4a570b05d3e8246cb99/src/TokenRepository.php#L110

Reply:
My bad! Thanks for pointing that 🙂